concent_api/core/utils.py

```diff
@@ -102,3 +104,16 @@ def hex_to_bytes_convert(client_public_key: str):
     key_bytes = decode_hex(client_public_key)
     assert len(key_bytes) == GOLEM_PUBLIC_KEY_LENGTH
     return key_bytes
+
+
+def extract_name_from_scene_file_path(scene_file_path):
+    for golem_resources_path in AVAILABLE_SCENE_FILE_PATHS:
+        if scene_file_path.startswith(golem_resources_path):
+            scene_file = scene_file_path[len(golem_resources_path):]
```

Comment:
This is misleading. `scene_file` and `scene_file_path` are both paths to a scene file. The difference is that one is a relative path within the archive and the other an absolute path within the docker image with Blender.

Reply:
Done